fetch.bs

```diff
@@ -3586,9 +3678,8 @@ optionally with a <i>recursive flag</i>, run these steps:
   <p>If <var>internalResponse</var>'s <a for=response>URL list</a> <a for=list>is empty</a>, then
   set it to a <a for=list>clone</a> of <var>request</var>'s <a for=request>URL list</a>.
 
-  <p class=note>A <a for=/>response</a>'s <a for=response>URL list</a> will typically be empty at
-  this point, unless it came from a service worker, in which case it will only be empty if it was
-  created through <a lt="Response()" constructor><code>new Response()</code></a>.
+  <p class=note>A <a for=/>response</a>'s <a for=response>URL list</a> can be empty when it's for
+  "<code>about</code>" URLs for example.
```

Comment:
This change seems a bit out of place? And the phrasing is a bit weird. If we keep it then I'd say

> A response's URL list can be empty (for example, when the response represents an `about:` URL).

Reply:
Done.

Reply:
The latest revision doesn't use @domenic's suggested phrasing.

Reply:
Sorry, it seems I dropped the edit during merging. Fixed.